topics/proteomics/tutorials/maxquant-msstats-dda-lfq/tutorial.md

```diff
@@ -56,17 +57,19 @@ The raw data is available via the [PRIDE repository](https://www.ebi.ac.uk/pride
 >
 >    {% snippet faqs/galaxy/histories_create_new.md %}
 >
-> 2. Import the FASTA database, annotation file and comparison matrix from [Zenodo](https://zenodo.org/record/4544529)
+> 2. Import the FASTA database, annotation file and comparison matrix from [Zenodo](https://zenodo.org/record/4896554)
```

Comment:
you could set `zenodo_link` in the top of the markdown, and then use `{{ page.zenodo_link }}` here, if you wanted to avoid updating the same 4 numbers every time.

Reply:
thank you for the hint @hexylena, I will keep it in mind for the next trainings, now I need to finally record the video ;)